### ISSUE TITLE
Show the Windows task bar in full-screen mode

### ISSUE THREAD

**Issue by BohrShaw** [kkoenig/wimproved.vim#19]
In a normal session, I prefer the Windows task bar be visible and anything else hidden.

Is this possible? Thanks!

**Comment by valacar** [comment]
If you're compiling this yourself, you can make a quick hack to do this.  Find the line `r = mi.rcMonitor;` in the `set_fullscreen()` function of the wimproved.c file, and change it to `r = mi.rcWork;`.

I doubt this would be good as the default behavior though, so changes in the vim script would need to be made as well.

**Comment by kkoenig** [comment]
I'm a big fan of minimal configuration options, but if this gets a few +1's I'll can add a g:wimproved_show_taskbar_in_fullscreen setting.

**Comment by BohrShaw** [comment]
@kkoenig Full screen ggk

**Comment by BohrShaw** [comment]
@kkoenig I prefer the current full screen mode which hides the task bar.

But I wish `:WToggleClean` behaves just like `:WToggleFullscreen` except that the task bar is visible.

**Comment by kkoenig** [comment]
@BohrShaw `:WToggleClean` does not modify window size, do you mean the menu bar inside of gVim?

**Comment by BohrShaw** [comment]
@kkoenig No, I don't mean the menu bar. And I didn't notice `:WToggleClean` does not modify window size as I always keep the gVim window maximized.
